Add purge osmosis script to docker server

amittar-lb opened the pull request:
The script erases all redundant labels from osmosis and then purge it
Copy /scripts folder to osmosis docker on Dockerfile

Execute script commands:
	ssh in to lightsrv1
	docker exec -it osmosis bash
	scripts/./purge_osmosis.sh

amittar-lb committed:
Add purge osmosis script to docker server

The script erases all redundant labels from osmosis and then purge it
Copy /scripts folder to osmosis docker on Dockerfile

Execute script commands:
	ssh in to lightsrv1
	docker exec -it osmosis bash
	scripts/./purge_osmosis.sh

yuval-lb commented on scripts/purge_osmosis.sh:
```diff
@@ -0,0 +1,17 @@
+echo finished erasing labels
+
+echo going to purge osmosis, this may take a while...
+/osmosis.bin purge --objectStoreRootPath=/data
```
looks good!
can we make some things configurable:
- osmosis bin path
- objectstore path

also the list, can it be a separate file? each label is a new line -> if this makes things very complicate then skip

amittar-lb replied:
Yes I can check how this things are done in bash and evaluate effort accordingly

amittar-lb committed:
Added command line parameters and Readme file

added labels.list that contains all labels to remove
added Readme.md file with usage and examples
added command line parameters:
	--help
	--labels_path <arg>
	--osmosis_bin_path <arg>
	--objectstore_path <arg>
